src/nvim/screen.c

```diff
@@ -7505,6 +7505,9 @@ void screen_resize(int width, int height)
   ++busy;
 
   Rows = height;
+  if (!ui_has(kUIMessages) && p_ch > Rows - min_rows() - 1) {
```

Comment:
Intentional that expressions are different? If not perhaps break out to a var.

Reply:
Oops. Just a mistake.